src/topp/QualityControl.cpp

```diff
@@ -451,71 +481,48 @@ class TOPPQualityControl : public TOPPBase
     }
     return files;
   }
-  
-  void fillConsensusPepIDMap_(vector<PeptideIdentification>& cpep_ids,
-                              const map<String, StringList>& identifier_to_msrunpath,
-                              multimap<String, PeptideIdentification*>& customID_to_cpepID) const
-  {
-    for (PeptideIdentification& cpep_id : cpep_ids)
-    {
-      if (!cpep_id.metaValueExists("spectrum_reference"))
-      {
-        throw Exception::MissingInformation(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION, "Spectrum reference missing at PeptideIdentification.");
-      }
-      const auto& ms_run_path = identifier_to_msrunpath.at(cpep_id.getIdentifier());
 
-      String UID; //< unique ID to identify the PepID
-      if (ms_run_path.size() == 1)
-      {
-        UID = ms_run_path[0] + cpep_id.getMetaValue("spectrum_reference").toString();
-      }
-      else if (cpep_id.metaValueExists("map_index"))
-      {
-        UID = cpep_id.getMetaValue("map_index").toString() + cpep_id.getMetaValue("spectrum_reference").toString();
-      }
-      else
+  void sortVectorOfPeptideIDsbyScore_(std::vector<PeptideIdentification> pep_ids)
```

Comment:
TODO: sorting consensus map hits by score seems rather standard. check if we have already some methods for that

Reply:
We looked again for it but we have not found such a function.